dev-scheme/chicken/chicken-5.2.0.recipe

```diff
@@ -0,0 +1,122 @@
+SUMMARY="A Scheme interpreter and native Scheme to C compiler"
+DESCRIPTION="CHICKEN is a compiler for the Scheme programming language. \
+CHICKEN produces portable and efficient C, supports almost all of the R5RS \
+Scheme language standard, and includes many enhancements and extensions. \
+CHICKEN runs on Linux, MacOS X, Windows, and many Unix flavours."
+HOMEPAGE="http://www.call-cc.org/"
+COPYRIGHT="2008-2020 The Chicken Team
+	2000-2007 Felix L. Winkelmann"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="http://code.call-cc.org/releases/$portVersion/chicken-$portVersion.tar.gz"
+CHECKSUM_SHA256="819149c8ce7303a9b381d3fdc1d5765c5f9ac4dee6f627d1652f47966a8780fa"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
```

Comment:
For now you can strike x86_gcc2 as a possible candidate, too much parse errors :)
```diff
- ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+ ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```